src/features/posts/components/user-label.tsx

```diff
@@ -0,0 +1,12 @@
+import { UserLabelSchema } from "../types/user-label"
+
+const userImg = "https://img.freepik.com/premium-photo/chita_827316-164.jpg"
+
+export const UserLabel = ({ user }: { user: UserLabelSchema }) => {
+    return(
+        <div className="w-full h-[100px] flex flex-row items-center justify-start">
+                <img src={user.image ? user.image : userImg} alt="post" className="w-[70px] h-[70px] m-5 object-cover rounded-[50%]" />
```

Comment:
Nie wiem czy to dobra praktyka, żeby każdy użytkownik jako profilowe miał czitke jeżeli nie ma swojego zdjęcia, ale dla mnie zajebiste